src/astUtils/creators.ts

```diff
@@ -1,52 +1,50 @@
-import { Position } from 'vscode-languageserver';
+import { Range } from 'vscode-languageserver';
 import { Token } from '../lexer/Token';
 import { TokenKind } from '../lexer/TokenKind';
 import { LiteralExpression, Expression, CallExpression, NamespacedVariableNameExpression, DottedGetExpression, VariableExpression } from '../parser/Expression';
-import { BrsType, BrsString, BrsInvalid, Int32, Float } from '../brsTypes';
-import util from '../util';
 
-export function createRange(pos: Position) {
-    return util.createRange(pos.line, pos.character, pos.line, pos.character);
-}
+export const interpolatedRange = Range.create(-1, -1, -1, -1);
 
-export function createToken<T extends TokenKind>(kind: T, pos: Position, text?: string, literal?: BrsType): Token & { kind: T } {
+export function createToken<T extends TokenKind>(kind: T, text?: string, range = interpolatedRange): Token & { kind: T } {
```

Comment:
Changed this to a range instead of a position, so a real range can be passed in if desired.